cloudify_agent/installer/__init__.py

```diff
@@ -49,7 +49,7 @@ def run_agent_command(self, command, execution_env=None):
         if execution_env is None:
             execution_env = {}
 
-        # TBD: Investigate why adding quotes don't work on linux
+        # TBD: Investigate why adding quotes doesn't work on linux
```

Comment:
Quotes don't work on linux because our `LinuxInstallerMixin` command is `/venv/bin/python /venv/bin/cfy-agent` (ie. python is the executable, and cfy-agent is the first argument) - and a shell doesn't allow commands that are quoted like `"executable arg" arg2`


Note: if someone was able to prove the `/venv/bin/python` prefix isn't necessary, and `/venv/bin/cfy-agent` is enough (ie. `cfy-agent` is executable in all cases - source or package install), then they would be able to remove the `/../bin/python` from `LinuxInstallerMixin`, and they could add quotes for linux as well.